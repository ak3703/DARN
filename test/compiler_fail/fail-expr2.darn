int main(){
	int a;
	float b;
	a = 1;
	b = 3.0;
	a + b;
}
