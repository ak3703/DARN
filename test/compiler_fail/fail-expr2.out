Fatal error: exception Failure("Illegal binary operator int + float in a + b")
